Phonebook.Backend/Phonebook.Backend.PictureService/Phonebook.Backend.PictureService/Controllers/EmployeePicturesController.cs

```diff
@@ -0,0 +1,149 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Cors;
+using Phonebook.Backend.PictureService.Helpers;
+
+namespace Phonebook.Backend.PictureService.Controllers
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    [Authorize]
+    [EnableCors("AllowDomainList")]
+    [Route("/")]
+    public class EmployeePictureController : Controller
+    {
+        /// <summary>
+        /// Upload an employee picture.
+        /// </summary>
+        /// <remarks>Content has to be formatted as "image/jpeg"</remarks>
+        /// See <see cref="UserController.WhoAmI()"/> to get the currently logged in user.
+        /// <param name="id">The id of the user.</param>
+        /// <param name="file">A "multipart"</param>
+        /// <returns>
+        /// A status code indicating the success of the action.
+        /// </returns>
+        /// <permission cref="System.Security.PermissionSet">Everyone can access this method.</permission>
+        /// <response code="200">Ok</response>
+        /// <response code="400">No Payload</response>
+        /// <response code="400">Content malformated</response>
+        /// <response code="401">User not authorized</response>
+        /// <response code="500">Internal Server Error</response>
+        [HttpPost("{id}")]
+        [RequestSizeLimit(2147483648)]

```

Comment:
Make this configurable via appsettings

Reply:
this could be a tricky one

Reply:
Not possible to solve now. Needs bigger changes. Will do later.